Mathlib/Analysis/MeanInequalities.lean

```diff
@@ -253,6 +267,58 @@ end Real
 
 end GeomMeanLEArithMean
 
+section HarmMeanLEGeomMean
+
+/-! ### HM-GM inequality -/
+
+namespace Real
+
+/-- **HM-GM inequality**: The harmonic mean is less than or equal to the geometric mean, weighted
+version for real-valued nonnegative functions. -/
+theorem harm_mean_le_geom_mean_weighted (w z : ι → ℝ) (hs : s.Nonempty) (hw : ∀ i ∈ s, 0 < w i)
```

Comment:
This should be implicit because `w` and `z` can be inferred from `hw` and `hz`. 

```suggestion
theorem harm_mean_le_geom_mean_weighted {w z : ι → ℝ} (hs : s.Nonempty) (hw : ∀ i ∈ s, 0 < w i)
```

Reply:
This is for consistency with all the other theorems in the file